.github/workflows/ci.yml

```diff
@@ -38,13 +49,12 @@ jobs:
           PHRASE: ${{ secrets.AUTOPARAMETERS_PASSPHRASE }}
         run: |
           set -vx
-          which pip
-          pip --version
-          python boots.py ensure
-          venv/Scripts/black --check --diff .
-          venv/Scripts/python download_fonts.py src/libs
-          venv/Scripts/genbuildinfo "src/epyq/_build.py"
-          venv/Scripts/pip freeze --all
+          poetry run buildui
+          poetry run python ./generate_versioning.py
+          .venv/Scripts/black --exclude=.\.venv --check --diff .
+          .venv/Scripts/python download_fonts.py src/libs
+          .venv/Scripts/genbuildinfo "src/epyq/_build.py"
+          .venv/Scripts/pip freeze --all
```

Comment:
for some reason `poetry run ...` didnt work when i changed them all to that so i kept .venv/ instead